cves/CVE-2018-13304.yml

```diff
@@ -90,32 +106,19 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: bd27a9364ca274ca97f1df6d984e88a0700fb235
-  note: ''
-- commit: ''
-  note: ''
+  note:
+
 vccs:
-- commit: 2aa9047486dbff12d9e040f917e5f799ed2fd78b
-  note: Identified by archeogit,
 - commit: f9d3841ae6147eaa51c57c574cd81e9ce9566e3a
   note: Identified by archeogit,
-- commit: 4d2858deac5213eaddfdc06f98379b6325d7b953
-  note: Identified by SZZUnleashed
-- commit: ca334dd14b957c4af7fcab5e97b6ed675c97caad
-  note: Identified by SZZUnleashed
-- commit: de6d9b6404bfd1c589799142da5a95428f146edd
-  note: Identified by SZZUnleashed
-- commit: 983e3246b7528e6ffc0cc347d54b86e161140fe6
-  note: Identified by SZZUnleashed
-- commit: ''
-  note: ''
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
 
   For the second round of reviewing, you will be giving a certain amount of
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes:
+upvotes: 3
```

Comment:
I give this 6 upvotes because I personally didn't know much about the encoding process before and I feel like you did a great job explaining it.